### PR TITLE
add additional BED interval information

### PR DESCRIPTION
**Pull request by jharenza**
#### Purpose

_Briefly describe your changes here._
new info I learned while preparing V5 data

#### Issue

_What GitHub issue does your pull request address?_

#### Pull review checklist

The PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ We required at least 60% of RNA-Seq reads mapped to the human reference or sampl
 #### SNV and INDEL calling
 
 We used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and VarDict.
-The same input interval BED files were used for both panel and WXS data.
+The same input interval BED files were used for both panel and WXS data and these were padded by 100 bp for all variant calling algorithm runs.
```
Comment (cansav09) [inline content/03.methods.md]: Does this mean +100bp was added on both sides of the range? Or +50bp on each side? Also what was the reasoning behind this padding? Did this come from a recommendation from one of the callers?

File: content/03.methods.md [hunk 2]
```diff
@@ -77,7 +77,8 @@ We required at least 60% of RNA-Seq reads mapped to the human reference or sampl
...
+The BED files for WGS were not padded for Mutect2 and Strelka2 runs, were padded by 300 bp for Lancet, and by 100 bp for VarDict.
```
Comment (cansav09) [inline content/03.methods.md]: Why were these padding window numbers chosen? Also, I have the same question about how this window padding is reported.

File: content/03.methods.md (hunk 1 above)
Comment (jharenza) [reply content/03.methods.md]: 100bp on each side. This did come from recommendations from the callers, with the exception of lancet - we tested multiple and took the best for precision and performance.

File: content/03.methods.md (hunk 2 above)
Comment (jharenza) [reply content/03.methods.md]: whoops, answered this above. Does that help?

Comment (cansav09) [reply content/03.methods.md]: Yes! It does. Can you add this information in the text?

Comment (jharenza) [reply content/03.methods.md]: done!

**Pull request merged**